src/iasm.c

```diff
@@ -413,6 +413,8 @@ struct OPND
     }
 };
 
+static OPND* const emptyOPND = (OPND*)(-1);
```

Comment:
Null seems like a good choice for a missing operand, what's the distinction to empty? Needs an explanatory comment.